src/python/WMCore/WMRuntime/Scripts/SetupCMSSWPset.py

```diff
@@ -728,89 +687,83 @@ def __call__(self):
 
             if socket.getfqdn().endswith("cern.ch"):
                 self.handleSpecialCERNMergeSettings(funcName)
-
         else:
+            self.logger.info("DEBUG: Now in the none scenario to load PSET")
             try:
                 self.loadPSet()
             except Exception as ex:
                 self.logger.exception("Error loading PSet:")
                 raise ex
 
+        dummyTweak = PSetTweak()
+        procScript = "edm_pset_tweak.py"
+        dummypsetTweakJson = os.path.join(self.stepSpace.location, "PSetTweak_dummy.json")
+        dummyTweak.persist(dummypsetTweakJson, formatting='simplejson')
+
+        cmd = "%s --input_pkl %s --output_pkl %s --json %s" % (
+            procScript,
+            os.path.join(self.stepSpace.location, self.configPickle),
+            os.path.join(self.stepSpace.location, "PSet_before.pkl"),
```

Comment:
What is this `PSet_before.pkl` for? If it's the original PSet and we keep it in the worker node, then it will be part of the logArchive tarball, which can increase its size significantly in case it has a large pileup dataset information (injected all the way from McM).

Reply:
As mentioned above, I will just remove this, as it is not needed.